src/rebar3_hex.erl

```diff
@@ -25,9 +25,7 @@ init(State) ->
                                                    rebar3_hex_key,
                                                    rebar3_hex_owner,
                                                    rebar3_hex_repo,
-                                                   rebar3_hex_docs,
```

Comment:
docs module is removed, will be re-added in another PR but won't handle generating docs and so forth